### DISCUSSION TITLE
OrgSwitcher: Sort list in alphabetical order

### DISCUSSION THREAD

**Discussion by pablozone** [grafana/grafana#38364] [Dashboards, Panels & Visualizations]
Hi can you please sort alphabetical list of organizations in Switch Organization window 

version 6.6.1

**Comment by papagian** [comment]
@pablozone 
Can you provide a non working example?
The organisations are sorted [by name](https://github.com/grafana/grafana/blob/master/pkg/services/sqlstore/user.go#L320).

**Comment by pablozone** [comment]
Added screens from grafana 6.5.2 which we use already, 
and the same from granfana 6.6.1
![list_grafana_6 5 2](https://user-images.githubusercontent.com/29774180/74719619-a9df9280-5234-11ea-98c1-edc89e1559ab.jpg)
![list_grafana_6 6 1](https://user-images.githubusercontent.com/29774180/74719630-b106a080-5234-11ea-93df-de63b8a0e01e.jpg)

**Comment by deathwarrior60** [comment]
#22163

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
